tests/end2end-tests/graphstorm-ec/test.sh

```diff
@@ -28,55 +28,55 @@ echo "Test GraphStorm edge classification"
 date
 
```

Comment:
There are a lot of duplicated of --part-config and --ip-config in these test commands. 

Maybe just leave as it is, and clean them up after the 0.1 release.

Reply:
Let's leave it to another PR to clean up tests.